README.md

```diff
@@ -1 +1,46 @@
-# Mobile-version
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Portfolio: mobile version.
+
+
+## Built With
+
+- HTML5
+- CSS
+- Flexbox and CSS Grid in order to make a structure in the html and look nice.
+
+## Getting Started
+
+- You must need to install npm packages (Linters such as Lighthouse, webhint and stylehintrc) in order to make work the project.
+
+**This is an example of how you may give instructions on setting up your project locally.**
+
+This is about a mobile version application, there is two parts so far, header and section in the middle wich l show works and about me.
+
+## Authors
+
+👤 **Author1**
```

Comment:
- [ ] [OPTIONAL] Great job replacing the placeholder social kinks with your social media links. I kindly suggest that you add your name(Authur name) here also instead